tests/js/jest.config.json

```diff
@@ -9,7 +9,11 @@
 	"moduleDirectories": [ "node_modules" ],
 	"moduleNameMapper": {
 		"@woocommerce/settings": "assets/js/settings/shared",
-		"@woocommerce/block-settings": "assets/js/settings/blocks"
+    "@woocommerce/block-settings": "assets/js/settings/blocks",
```

Comment:
Small alignment issue in these lines.

Reply:
Dunno what was going on here since it looked fine in my IDE so used vscode to auto-format the entire file (see c37e736)

Reply:
> Dunno what was going on here since it looked fine in my IDE so used vscode to auto-format the entire file (see c37e736)

That changed all lines to use spaces instead of tabs, You can run `npm run reformat-files` to let Prettier format the file. Is it ok to use prettier format or is there any reason you prefer spaces to tabs in JSON files?

Reply:
hmm, vscode must not be configured correctly for my system though, is our commit-hook prettier implementation configured to format json files too?  Should it be?

Reply:
Problem was editor config 9792513